app/views/admin/admins/show.html.erb

```diff
@@ -0,0 +1,53 @@
+<<h1>Dashboard</h1>
+<<h3>Packaged Orders:</h3>
+<<div class="orders-index">
+  <% @packaged_orders.each do |order| %>
+  <<section id="order-id-<%= order.id %>">
+    <%= link_to order.user.name, admin_user_path(order.user) %>
+    <<ul>
+      <<li>Order Id: <%= order.id %></li>
+      <<li>Order Status: <%= order.status %></li>
+      <<li>Created At: <%= order.created_at %></li>
```

Comment:
Try a save_and_open_page... Elsewhere on the site we're doing `Date.strptime(order.created_at.to_s)`.

Reply:
Same issue for each section